about.html

```diff
@@ -27,7 +27,7 @@
           class="w-full grid grid-cols-2 items-center justify-between hidden md:flex"
         >
           <a href="index.html">
-            <img width="200" src="./images/LOGO.svg" alt="" />
+            <img width="200" src="./images/LOGO.svg" alt="ex-investor logo" />
           </a>
           <ul class="flex gap-6 justify-end">
             <li>
```

Comment:
- [ ] [OPTIONAL] I would suggest to update your About pages mobile menu, so that there is a way for a user to go back to home page from the about page, currently the nav links are :point_down: 
![image](https://user-images.githubusercontent.com/97521557/225693976-d91fc5f3-012f-4010-8268-31a9143419dd.png)

you should update to `<li><a href="./index.html">Home</a></li>` this will make sure the user can go back to home page from the about page easily